.github/workflows/scorecards.yml

```diff
@@ -35,7 +36,7 @@ jobs:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@34bc6b110eae5d331dde534cd371be23a51be59d # v1.1.1
+        uses: ossf/scorecard-action@3e15ea8318eee9b333819ec77a36aca8d39df13e # v1.1.1
```

Comment:
shouldn't we update the version comment too?

Reply:
kay